docs/hsi.xml

```diff
@@ -393,6 +393,22 @@
       </note>
     </refsect3>
 
+    <refsect3 id="org.fwupd.hsi.Spi.Descriptor">
+      <title>Read-only SPI Descriptor</title>
+      <para>
+        The SPI descriptor must always be read only from all other IFD regions.
```

Comment:
This is written to reference "Intel flash descriptor" but I think you want it in HSI as a generic requirement, it will vary from different implementations, no?

Reply:
Fixed.